### PR TITLE
add mastering ipython book

### PR DESCRIPTION
**Pull request by minrk**
cc @fperez @Carreau @ellisonbg

### CODE REVIEW

**Comment by Carreau** [comment]
That would add "Mastering IPython 4.0" a second time on the website :-)

**Comment by Carreau** [comment]
Though we can merge your changes into the one I did yesterday.

File: books.rst [hunk 1]
```diff
@@ -2,6 +2,34 @@
...
+
+.. image:: _static/mastering_ipython_40.png
+    :width: 200px
+    :alt: Cover of "Mastering IPtyhon 4.9"
+    :target: _static/mastering_ipython_40.png
+
+* `Mastering IPython 4.0 <https://www.packtpub.com/big-data-and-business-intelligence/mastering-ipython-40>`_
+* by Thomas Bitterman
+* Packt Publishing
+* Code Available under MIT Liscence `on GitHub PacktPublishing Organisation <https://github.com/PacktPublishing/Mastering-IPython-4>`_
```
Comment (blink1073) [inline books.rst]: License*

**Comment by minrk** [comment]
Whoops, didn't see the earlier PR. I made this one yesterday but forgot to open it. Thanks for the fix, should be free of conflicts, now.

**Pull request merged**